src/main/scala/uk/gov/hmrc/mongo/ReactiveRepository.scala

```diff
@@ -20,8 +19,9 @@ abstract class ReactiveRepository[A <: Any, ID <: Any](collectionName: String,
                                                        idFormat: Format[ID] = ReactiveMongoFormats.objectIdFormats,
                                                        mc: Option[JSONCollection] = None)
                                                       (implicit manifest: Manifest[A], mid: Manifest[ID])
-  extends Repository[A, ID] with Indexes with ImplicitBSONHandlers {
+  extends Repository[A, ID] with Indexes {
```

Comment:
We use this Repository trait in simple-reactive mongo right? We should ensure we add the ImplicitBSONHandlers in there